extensions/schema-compare/src/test/schemaCompare.test.ts

```diff
@@ -26,7 +26,7 @@ before(async function (): Promise<void> {
 	testContext = createContext();
 });
 describe('SchemaCompareDialog.openDialog', function (): void {
-	beforeEach(() => {
+	before(() => {
```

Comment:
just confirming: no need for a mock reset in beforeEach like you did below?

Reply:
mockExtensionContext object isn't getting updated anywhere in the test or the code, hence I felt that beforeEach wouldn't be required.
Let me know if you feel otherwise.